cuda/Main.cu

```diff
@@ -99,5 +101,11 @@ int main(int argc, char *argv[]) {
         int is_invalid_result =
                 print_results(in, mype, omp_end - omp_start, nprocs, verification);
 
-        return is_invalid_result;
+	printf("host_to_device_ms,kernel_ms,device_to_host_ms\n");
```

Comment:
If we want to keep track of the total time for all the kernel iterations we should also add a column indicating the number of iterations to easily calculate the average later.

Reply:
Will do